docs/posts/classification-metrics-on-the-backend/index.qmd

```diff
@@ -0,0 +1,210 @@
+---
+title: "Classification metrics on the backend"
+author: "Tyler White"
+date: "2024-11-15"
```

Comment:
```suggestion
date: "2024-11-18"
```

I always forget to update the date, so leaving this out as a reminder to adjust this
when we are ready to merge. (Not saying it needs to be 2024-11-18 😄)